woocommerce/woocommerce-gutenberg-products-block
# basic e2e test for instantiating All Reviews block

Part of #2395

This PR adds a basic e2e test for instantiating an All Reviews block. Note the code was pulled directly from #2756 PR, splitting those tests off into separate PRs (divide and conquer).

Credit to @senadir for all the code in this PR :)

### How to test the changes in this Pull Request:

1. Clone this branch.
2. Build e2e container infra stuff `npm run build:e2e-test`
3. Start wp-env containers - tests are run against the wp tests container `npm run wp-env start`
4. Run e2e tests `npm run test:e2e`

Tests should all pass, in particular new `All Reviews` test should pass.

Also the travis tests on this PR should succeed and include the `All Reviews` block test.
